### ISSUE TITLE
Design JSON-LD output 

### ISSUE THREAD

**Issue by bertspaan** [erfgoed-en-locatie/histograph#19]
GeoConcepts/Subgraphs and PITs should be easy.

For relations, use the following: 
http://stackoverflow.com/questions/23141875/how-to-describe-the-predicate-edge-of-a-triple-in-json-ld

**Comment by bertspaan** [comment]
Example:

``` json
{
  "@context": {
    "@base": "http://erfgeo.nl/hg/",
    "@vocab": "http://erfgeo.nl/",
    "hasBeginning": "time:hasBeginning",
    "features": "_:features",
    "source": {
      "@id": "source",
      "@type": "@id"
    },
        "uri": {
      "@id": "owl:sameAs",      
      "@type": "@id"      
    },
    "relations": "_:",
    "properties": "_:properties",
    "pits": "_:pits",  
    "name": "dc:title",
    "type": "hg:type",
    "geometryIndex": null,
    "geometry": null
  },
  "type": "FeatureCollection",
  "features": [
    {
      "properties": {
         "type": "hg:Place",
        "pits": [
          {
            "@id": "tgn/7006858",           
            "geometryIndex": 0,
            "hgid": "tgn/7006858", 
            "name": "Weert",
            "relations": [
              {
                        "@id": "tgn/7006858",
              "hg:conceptIdentical": [{"@id":"bag/weert"}]
              }
            ],
            "source": "tgn",
            "uri": "http://vocab.getty.edu/tgn/7006858",
            "hasBeginning": "2014-12-31",
            "type": "hg:Place"
          }
        ]
      }
    }
  ]
}
```

**Comment by bertspaan** [comment]
3fb6a84d025f2983a2047d983652ab0553b47cf9 closes #19
